src/umpire/strategy/DynamicPool.cpp

```diff
@@ -153,9 +158,19 @@ void* DynamicPool::allocateFromResource(std::size_t bytes)
       throw;
     }
   }
+
+  // Add to count
+  m_actual_bytes += bytes;
+
```

Comment:
Was this moved from someplace?  Or was it previously not be accounted for correctly?